### DISCUSSION TITLE
How do I add an external converter file in zigbee2mqtt LXC?

### DISCUSSION THREAD

**Discussion by vdiogo** [tteck/Proxmox#2568] [Help Needed] [answered]
I want to add a Zigbee device which is not supported by z2m, so would like to add the .js file in z2m config folder.
How do I achieve this please?

**Comment by tteck** [comment]
https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html

**Reply by vdiogo** [reply]
Thanks, I did saw that prior raising this Q, but my question is, considering z2m is installed in a container, how do I get an external file to be put on the same folder as the config.yaml? 

I'm not familiar with Linux and such, so any guidance would be welcomed

**Reply by tteck** [reply]
The instructions on the website shows the path to the configuration.yaml

**Reply by vdiogo** [reply]
I read `save it next to the Zigbee2MQTT configuration.yaml` but how to I get access to it considering it's installed on a container? 
I have the .js file to be used as external converter, but how do I put it there? 
I also understand I can reference a url from where the .js file can be fetched - trigger OTA update - but after multiple browsing on the internet, I can't find my way around it on how to list the folder, access it, and eventually edit the config.yaml to reference OTA alternative url

**Reply by tteck** [reply, marked as answer]
I'd suggest installing File Browser in the container as a way to have a graphical user interface (GUI) for uploading, downloading, and editing files.

**Reply by vdiogo** [reply]
Awesome, great guidance!! 
Did it (and of course used your script) :)
